examples/tests/sdk_tests/api/post.test.w

```diff
@@ -19,15 +20,14 @@ api.post("/path", inflight (req: cloud.ApiRequest): cloud.ApiResponse => {
 
 
 test "http.post and http.fetch can preform a call to an api" {
-  let url = api.url + "/path";
-  let response: http.Response = http.post(url, headers: { "content-type" => "application/json" }, body: Json.stringify(body));
-  let fetchResponse: http.Response = http.post(url, method: http.HttpMethod.POST, headers: { "content-type" => "application/json" }, body: Json.stringify(body));
+  let response: http.Response = http.post(api.url, headers: { "content-type" => "application/json" }, body: Json.stringify(body));
+  let fetchResponse: http.Response = http.post(api.url, method: http.HttpMethod.POST, headers: { "content-type" => "application/json" }, body: Json.stringify(body));
 
-  assert(response.body == Json.stringify(body));
-  assert(response.status == 200);
-  assert(response.url == url);
+  expect.equal(response.body , Json.stringify(body));
+  expect.equal(response.status , 200);
+  expect.match(response.url , api.url);
```

Comment:
this is match since the response url in the sim includes an extra "/" at the end